zh-Hans.lproj/Shared.strings

```diff
@@ -0,0 +1,39 @@
+/*
+  Localizable.strings
+  Redacted
+
+  Created by Sam Soffes on 4/14/15.
+  Copyright © 2015–2017 Nothing Magical Inc. All rights reserved.
+*/
+
+/* Modes */
+"MODE" = "模式";
+"PIXELATE" = "打码";
+"BLUR" = "模糊";
+"BLACK_BAR" = "黑条";
+
+/* Actions */
+"SHARE" = "分享";
+"CLEAR" = "清除";
+"CLEAR_IMAGE" = "清除图片";
+"SAVE" = "保存";
+"EXPORT" = "导出…";
+
+/* Edit menu */
+"COPY_IMAGE" = "复制图片";
+"PASTE_IMAGE" = "粘贴图片";
+"SELECT_ALL_REDACTIONS" = "选择所有 Redactions";
```

Comment:
Is there not a good Chinese word for this? I'm fine to leave it in English if you think that's best. I'm not treating it like the app name but just the generic word so feel free to translate it if you want to!